keps/sig-api-machinery/2876-crd-validation-expression-language/README.md

```diff
@@ -475,24 +500,49 @@ coverage of interactions in these dimensions:
 
 Types:
 
-| OpenAPIv3 type                          | CEL type                       |
-| --------------------------------------- | ------------------------------ |
-| 'object' with Properties                | object / "message type"        |
-| 'object' with AdditionalProperties      | map                            |
-| 'array                                  | list                           |
-| 'array' with x-kubernetes-list-type=map | list with map based Equality & unique key guarantees   |
-| 'array' with x-kubernetes-list-type=set | list with set based Equality & unique entry guarantees |
-| 'boolean'                               | boolean                        |
-| 'number' (all formats)                  | double                         |
-| 'integer' (all formats)                 | int (64)                       |
-| 'null'                                  | null                           |
-| 'string'                                | string                         |
-| 'string' with format=byte               | bytes                          |
-| 'string' with format=binary             | bytes                          |
-| 'string' with format=date               | timestamp (protobuf.Timestamp) |
-| 'string' with format=datetime           | timestamp (protobuf.Timestamp) |
-
-xref: [CEL types](https://github.com/google/cel-spec/blob/master/doc/langdef.md#values), [OpenAPI types](https://swagger.io/specification/#data-types).
+| OpenAPIv3 type                                     | CEL type                                                                                                                     |
+| -------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| 'object' with Properties                           | object / "message type"                                                                                                      |
```

Comment:
What does `type(<object>)` produce as an identifier? Might be helpful to have an example here.

Reply:
Added.

It's `selfType<uniqueNumber>.path.to.object.from.self`. We are deliberately denying developers a stable type in CEL expressions right now since we're not supporting type construction and we're trying keep the surface area that developers can depend on to a minimum.